R/fit.R

```diff
@@ -684,7 +683,7 @@ CmdStanMCMC <- R6::R6Class(
       }
       matching_res <- matching_variables(variables, private$metadata_$model_params)
       if (length(matching_res$not_found)) {
-        stop("Can't find the following variable(s) in the sampling output: ",
+        stop("Can't find the following variable(s) in the output: ",
```

Comment:
I removed the "sampling" from this error message so the messages are consistent with those in read_cmdstan_csv.